modules/apps/transfer/keeper/relay_forwarding_test.go

```diff
@@ -223,6 +223,90 @@ func (suite *KeeperTestSuite) TestSuccessfulForward() {
 	suite.Require().NoError(err)
 }
 
+// TestSuccessfulPathForwarding tests that a packet is successfully forwarded with a non-Cosmos account address.
+// The test stops before verifying the final receive, because we don't have a non-cosmos chain to test with.
+func (suite *KeeperTestSuite) TestSuccessfulForwardWithNonCosmosAccAddress() {
+	/*
+		Given the following topology:
+		chain A (channel 0) -> (channel-0) chain B (channel-1) -> (channel-0) chain C
+		stake                  transfer/channel-0/stake           transfer/channel-0/transfer/channel-0/stake
+		We want to trigger:
+			1. A sends B over channel-0.
+			2. Receive on B.
+				2.1 B sends C over channel-1
+			3. Receive on C.
+		At this point we want to assert:
+			A: packet gets relayed properly with final receiver intact
+			B: packet arrives and is forwarded with final receiver intact
+			C: no assertions as we don't have a non-cosmos chain to test with, so this would fail here
+	*/
+
+	amount := sdkmath.NewInt(100)
+
+	path1 := ibctesting.NewTransferPath(suite.chainA, suite.chainB)
+	path1.Setup()
+
+	path2 := ibctesting.NewTransferPath(suite.chainB, suite.chainC)
+	path2.Setup()
```

Comment:
You could do this like in the other tests:

```suggestion
	pathAtoB, pathBtoC := suite.setupForwardingPaths()
```

Reply:
Oh yeah, that is much better obviously. Forgot about that one. Will fix! :)